hw/hw03/switcherblades/homework3.py

```diff
@@ -0,0 +1,27 @@
+import shutil
+txt_path = r'E:\workspace\pycharm_projects\pythonFundamental\UA-1013.Python\hw\hw01\switcherblades\Zen.txt'
+dst_path = r'E:\workspace\pycharm_projects\pythonFundamental\UA-1013.Python\hw\hw03\switcherblades'
+shutil.copy(txt_path, dst_path)
+with open("Zen.txt") as file:
+    text = file.read()
```

Comment:
relative paths should be used
there was no need to use files here
you just had to declare the string